source/client/benchmark_client_impl.h

```diff
@@ -108,7 +109,8 @@ class BenchmarkClientHttpImpl : public BenchmarkClient,
                           Envoy::Tracing::HttpTracerSharedPtr& http_tracer,
                           absl::string_view cluster_name, RequestGenerator request_generator,
                           const bool provide_resource_backpressure,
-                          absl::string_view latency_response_header_name);
+                          absl::string_view latency_response_header_name,
+                          std::vector<UserDefinedOutputPluginPtr> user_defined_output_plugins);
```

Comment:
Should this be passed by reference?

Reply:
Actually, here we're moving rather than passing by reference, so this is a rare case where "passing by value" is the correct/optimal thing to do.

Reply:
Ack.